src/API/WP/NotificationsService.php

```diff
@@ -4,11 +4,13 @@
 namespace Automattic\WooCommerce\GoogleListingsAndAds\API\WP;
 
 use Automattic\Jetpack\Connection\Client;
+use Automattic\WooCommerce\GoogleListingsAndAds\HelperTraits\SyncTrait;
```

Comment:
This is now the only location that SyncTrait is used, so it's not really sharing traits between classes. But I'm ok with leaving it as is.

Reply:
I will rectactor it in further PRs as also this implies we can use constants for the Data Types